arlo-client/src/components/CreateAudit.test.tsx

```diff
@@ -20,7 +20,7 @@ const checkAndToastMock: jest.SpyInstance<
 checkAndToastMock.mockReturnValue(false)
 
 const routeProps: RouteComponentProps<ICreateAuditParams> = routerTestProps(
-  '/election/:electionId',
+  '/audit/:electionId',
```

Comment:
wait, did i miss a URL change? I'm not seeing it

Reply:
or is this just for testing? If so it's a tad confusing.

Reply:
https://github.com/votingworks/arlo/pull/464/files#diff-c4fc915a5c89a9458ba5320c7f2be3f8R72-R73

the single-jurisdiction flow is going under `/audit` so we can use routing to know which flow we're supposed to render